angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.stories.ts

```diff
@@ -261,6 +263,18 @@ export const defaultStory = () => ({
         }
       ]"
     >
+      <sprk-stack
+        additional-disclaimer-slot
+        itemSpacing="large"
+        sprkStackItem
+      >
+        <p
+          sprkStackItem
+          class="sprk-b-TypeBodyFour sprk-c-Footer__text"
```

Comment:
hey @RVMendoza, looks like you forgot the sprkText directive here.

Reply:
done!